calendar/page.js

```diff
@@ -88,8 +88,11 @@ class CalendarHandler {
       this.calendar.updateEvent(record.id, CALENDAR_NAME, {backgroundColor: CalendarHandler._selectedColor});
       this._selectedRecordId = record.id;
       this.calendar.setDate(record.startDate);
+      updateUIAfterNavigation();
       if (this.calendar.getViewName() !== 'month') {
-        //Scoll to the middle of the event, if it's not month view
+        //Scroll to the middle of the event if it's not month view.
```

Comment:
Thanks! Could you please add a space after the `//`s, to match the bulk of Grist source code.